Xamarin.Forms.Platform.iOS/DisposeHelpers.cs

```diff
@@ -25,20 +25,25 @@ internal static void DisposeModalAndChildRenderers(this Element view)
 				}
 			}
 
-			if (view is VisualElement visualElement)
+			if (view is VisualElement visualElement
+#if __MOBILE__
+			  )
+#else
+			    && !(view is NavigationPage))
+#endif
```

Comment:
This is the main change and more symptomatic than actually a fix. I'd be keen to know if there's a better way to fix in and if so how.